src/lib.rs

```diff
@@ -44,7 +49,7 @@ fn get_identity(account_name: impl AsRef<Path>) -> Result<BasicIdentity> {
 /// mkdir -p ~/.config/dfx/identity/
 /// cp -Rn ./identity/.config/dfx/identity/* ~/.config/dfx/identity/
 /// ```
-pub async fn get_agent(name: impl AsRef<Path>, url: Option<&str>) -> Result<Agent> {
+pub async fn get_agent(name: impl Into<&str>, url: Option<&str>) -> Result<Agent> {
```

Comment:
use `Path` so it works with the `get_identity` suggestion
```suggestion
pub async fn get_agent(name: impl AsRef<Path>, url: Option<&str>) -> Result<Agent> {
```

Reply:
https://infinityswap.slack.com/archives/C02P69J279T/p1648709894710999?thread_ts=1648705979.062629&cid=C02P69J279T

Reply:
:+1: